indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -138,3 +139,17 @@ class RHMenuEntries(RHEditingManagementBase):
     def _process(self):
         menu_entries = named_objects_from_signal(signals.menu.items.send('event-editing-sidemenu', event=self.event))
         return EditingMenuItemSchema(many=True).jsonify(menu_entries.values())
+
+
+class RHEditableTypes(RHEditingManagementBase):
+    def _process_GET(self):
+        return jsonify(editing_settings.get(self.event, 'editable_types').items())
+
+    @use_kwargs(EditableTypeArgs)
+    def _process_POST(self, editable_types):
+        new_editable_types = editing_settings.get(self.event, 'editable_types')
+        enabled = [t.name for t in editable_types]
+        for key in new_editable_types:
+            new_editable_types[key] = key in enabled
+        editing_settings.set(self.event, 'editable_types', new_editable_types)
```

Comment:
Can't this whole block be replaced with this?

```python
editing_settings.set(self.event, 'editable_types', editable_types)
```

or if you want to keep a dict instead of storing a list of enabled types in the settings:

```python
types_config = {t.name: t.name in editable_types for t in EditableType}
editing_settings.set(self.event, 'editable_types', types_config)
```